src/idl_parser.cpp

```diff
@@ -1763,6 +1772,21 @@ Namespace *Parser::UniqueNamespace(Namespace *ns) {
   return ns;
 }
 
+std::string Parser::GetNameFromFullQualifiedName(std::string fullQualifiedName)
+{
+  Namespace *ns = new Namespace();
+
```

Comment:
no empty lines needed, unless function is really long.

Reply:
fixed.